### DISCUSSION TITLE
Form.List combined with >= 5.6.0 Collapsable

### DISCUSSION THREAD

**Discussion by roman-yu-yoc** [ant-design/ant-design#43488] [General]
### What problem does this feature solve?

Guys, I'm pretty sure that you already thought about this, but I just might as well address my concern here...

So Docs say that version of antd higher 5.6.* will no longer support JSX way of adding Collapse.Panel.

I'm working on form and I have Form.List component, I'm not really sure what would be the right way to actually combine them both there using new way of working with Collapsable.

Please advise! Thank you!

### What does the proposed API look like?

My suggestion would be not to deprecate old way of using Collapse component

**Comment by Slab3** [comment]
That's still pretty simple to use Form.List with 'items', here's example:
```
<Form.List name="items">
  {(fields, { add, remove }) => (
    <div className="flex-wrapper">
      <Collapse
        accordion
        items={fields.map((field, index) => ({
          key: field.key,
          label: `Item ${index + 1}`,
          children: <ItemRow field={field} index={index} />,
          extra: (<Button onClick={() => remove(field.name)} icon={<DeleteOutlined />} />),
        }))}
      />
      <Button
        type="default"
        onClick={() => add()}
        icon={<PlusOutlined />}
        className={styles.addBtn}
      >
        Add item
      </Button>
    </div>
  )}
</Form.List>
```
